### PR TITLE
feat(homepage): refactor homepage tabs into separate pages

### PR DESCRIPTION
**Pull request by benprotheroe**
### Job Story

**When** accessing the homepage
**I want to load the page without a content shift**
**So that the user has a clean experience of loading**.

### Implementation details

This happens because the homepage tabs rely on javascript which is loaded with the bundle. 

Switching from tabs to separate pages will take advantage of static rendering and thereby improve page performance

### Testing

go to:

https://deploy-preview-2747--oak-web-application.netlify.app/
https://deploy-preview-2747--oak-web-application.netlify.app/pupils
https://deploy-preview-2747--oak-web-application.netlify.app/teachers
https://deploy-preview-2747--oak-web-application.netlify.app/curriculum
https://deploy-preview-2747--oak-web-application.netlify.app/ai

use links to switch between these pages

go to:

https://deploy-preview-2747--oak-web-application.netlify.app/#pupils
https://deploy-preview-2747--oak-web-application.netlify.app/#teachers
https://deploy-preview-2747--oak-web-application.netlify.app/#curriculum
https://deploy-preview-2747--oak-web-application.netlify.app/#ai

you should be redirected

### Acceptance criteria

- [x]  page loads without glitch
- [x]  tabs rendered as separate pages
- [x]  tests updated
- [x]  links are accessible
- [x]  old anchor pages redirected

### Before performance

image

### After performance

image

### CODE REVIEW

File: src/pages/index.tsx [hunk 1]
```diff
@@ -1,166 +1,28 @@
...
-    } else if (current === undefined) {
-      // default value when first loading the page or invalid hash
-      setActiveTab("teachers");
+    // clientside redirect for old tabs implementation
+    if (window.location.href.includes("#pupils")) {
+      router.push("/pupils");
+    } else if (window.location.href.includes("#curriculum")) {
+      router.push("/curriculum");
+    } else if (window.location.href.includes("#ai")) {
+      router.push("/ai");
```
Comment (njblore) [inline src/pages/index.tsx:20]: would adding these as redirects in the next.config not work?

File: src/components/GenericPagesComponents/HomePageTabImageNav/HomePageTabImageNav.tsx [hunk 2]
```diff
@@ -3,16 +3,21 @@ import { OakFlex } from "@oaknational/oak-components";
...
   ...flexProps
 }: FlexProps & {
   current: HomePageTab | undefined;
-  setCurrent: (tab: HomePageTab) => void;
 }) => {
+  const setCurrent = (tab: HomePageTab) => {
+    if (tab === current) {
+      return;
+    }
+    window.location.href = `/${tab}`;
```
Comment (njblore) [inline src/components/GenericPagesComponents/HomePageTabImageNav/HomePageTabImageNav.tsx:19]: can we change the buttons to links to the pages now?

Comment (kimon-satan) [reply src/components/GenericPagesComponents/HomePageTabImageNav/HomePageTabImageNav.tsx]: alas not without completely rewriting these janky buttons. Postponed to another ticket

File: src/pages/index.tsx (hunk 1 above)
Comment (kimon-satan) [reply src/pages/index.tsx:40]: nope . unfortunately #s aren't recognised by next config. This is the only way to do it on pages router

Comment (njblore) [reply src/pages/index.tsx:40]: ahh I see, that's annoying 🫠

**Review by njblore — APPROVED** [review]

**Pull request merged**